themes/rhds/layouts/partials/eddl/page-load-completed.html

```diff
@@ -0,0 +1,4 @@
+<script>
```

Comment:
This event triggers the Page Load Started event to actually fire off.

Ref: https://github.com/searchdiscovery/redhat-datalayer/blob/main/EDDL/global-datalayer.md#page-load-completed